libs/ui/src/theme/HausThemeContext.tsx

```diff
@@ -51,7 +51,7 @@ export const HausThemeProvider: FunctionComponent<{
   return (
     <HausThemeContext.Provider value={{ theme, setTheme, toggleLightDark }}>
       <ThemeProvider theme={theme}>
-        <GlobalStyles />
+        <GlobalStyles theme={theme} />
```

Comment:
Does this relate to what you mentioned about needing to pass in theme values into the GlobalStlyes?

Reply:
Actually don't think this is needed.